File: Lehrverwaltung/client/src/app/core/models/run/RunModel.ts
export class RunModel {
  username: string;

  date: string;

  runtime: number;

  name: string;
  data: string;
  type: string;

  constructor(username: string, date: string, runtime: number, name: string, data: string, type: string) {
    this.username = username;
    this.date = date;
    this.runtime = runtime;
    this.name = name;
    this.data = data;
    this.type = type;
  }
}

File: Lehrverwaltung/server/src/main/java/tie/lehrlinge/lehrverwaltung/server/model/RunModel.java
package tie.lehrlinge.lehrverwaltung.server.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class RunModel {

    private String username;

    private String date;

    private int runtime;
    private String name;
    private byte[] data;
    private String type;
}
